.gitignore

```diff
@@ -2,3 +2,4 @@
 /node_modules
```

Comment:
i think you should remove the first `/` or replace it with `./` because that says from the root